fiona/__init__.py

```diff
@@ -90,7 +90,7 @@
 
     see https://github.com/Toblerity/Fiona/issues/851
     """
-    if platform.system() == 'Windows' and (3, 8) <= sys.version_info:
+    if platform.system() == 'Windows':
```

Comment:
I think it should not hurt to add the same directories for all versions of Python if the import of ogrext fails